api/lib/reducers/cosmosV3-reducers.js

```diff
@@ -107,16 +104,13 @@ function transactionReducerV2(network, transaction, reducers) {
         return coinReducer(coin, coinLookup, network)
       })
     } else {
-      fees = transaction.tx.auth_info.fee.amount.map(fee => {
-        const coinLookup = network.getCoinLookup(
-          network,
-          fee.denom
-        )
+      fees = transaction.tx.auth_info.fee.amount.map((fee) => {
+        const coinLookup = network.getCoinLookup(network, fee.denom)
         return coinReducer(fee, coinLookup, network)
       })
     }
     // We do display only the transactions we support in Lunie
-    const filteredMessages = transaction.tx.body.messages.filter(
+    const filteredMessages = transaction.tx.value.msg.filter(
```

Comment:
🙈  the truth is, this is cosmos hub 4 but the akash has still the old format here

Reply:
But I checked and Akash Testnet has the new, the `value.msg`. What is happening?

Reply:
no
that is the old format

Reply:
Did you try Akash-testnet with this in?

Reply:
OK, of course, silly question from me. It is working great of course. Amazing!